### PR TITLE
New accounts

### LINKED ISSUE #1151: Port `accounts` app templates to Django template syntax
We want to simplify the CiviWiki project by removing JavaScript SPA code written in Backbone.js and using Django templates instead. We will work through the Backbone templates in the `accounts` app to convert the template syntax from Backbone to Django. We want to maintain the visual styling. Below is a checklist of the relevant Backbone templates to be ported to complete this task.

Note: Backbone markup can be found between `{% verbatim %}` and `{% endverbatim %}` template tags as well as within `{% block backbone_template %}`.

## Task

Port the following templates to Django template syntax and ensure they render correctly via the relevant Django view. 

**Note:** all template baths below start within the `accounts/template/accounts/` directory, for brevity.

- [x] #1199
- [x] complete the following for `settings.html`
    - [x] create view for "Save Changes" form submission
    - [x] create view for "delete account" submission
    - [x] use vanilla JS confirm for "delete account" confirmation dialogue
- [x] convert Backbone view models to Django view/model in the following templates
    - [ ] `feed.html` - moved to `threads` app for follow-up work
    - [x] `settings.html`
- [x] ensure the following properties exist on the `User` or `Profile` model
    - [x]  `civis`
    - [x] `followers`
    - [x] `following`
    - [x] `categories`
    - [x] `threads`
    - [ ] `drafts`
- [x] convert API views to Django views
    - [x] `edit_user`
    - [x] `delete_user`
    - [x] `upload_profile_image`
    - [x] `clear_profile_image`
    - [x] `request_follow`
    - [x] `request_unfollow`
    - [x] `edit_user_categories`
- [x] simplify accounts views so there is only one `ProfileEdit` view
- [x] remove `ProfileActivationView`
- [x] remove `ProfileSetupView`
- [x] delete `user_setup.html` and any related routing logic (such as where we force the user through the setup wizard)
    - for simplicity, let's deprecate the setup wizard in favor of `update_settings.html`
- [ ] optionally, add `translate` tags for all accounts UI texts

### LINKED ISSUE #1199: port `account.html` to Django template syntax
The file `accounts/templates/accounts/account.html` contains several `varbatim` sections that render Backbone content. We want to replace the Backbone content with Django template syntax. 

- [ ] move all elements from `block backbone_template` to `block content`
- [ ] change the Backbone code to use Django template syntax
- [ ] Ensure that all of the following code is replaced by Django template syntax:

account.html - each verbatim section needs to be converted to Django template syntax and the view should provide necessary context data
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L14-L89
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L92-L102
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L105-L139'
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L142-L168
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L171-L176
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L195-L209
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L224-L241
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L246-L274
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L281-L295
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/8ced00f932235b1ad472dcae5ecc6d0b9525bcec/project/accounts/templates/accounts/account.html#L352-L369

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Closes #1199
  
## Description
replaces the update-settings.html and removes settings.html, mnimalistic html changes.

### CODE REVIEW

**Comment by brylie** [comment]
I'm converting this to a draft while it is work-in-progress.

**Comment by AbhijithGanesh** [comment]
@gorkemarslan What should I change, I am a bit confused actually!😅

**Comment by gorkemarslan** [comment]
@AbhijithGanesh, why did you revert your initial changes? 😅 

@brylie wants you to create new views. I guess this is one of the missing parts.

Edit: I didn't completely review your first changes but they seem ok with me. At which point do you need clarification? Are you confused by the template part? Apparently, you did well it. Again, I didn't try to run your changes.

Of course, we are going to think of views part together and find a way to solve that.

**Comment by AbhijithGanesh** [comment]
My doubt is, are we going to make new views for `Accounts` ? I have restructured the templates I reckon.

**Comment by brylie** [comment]
> @brylie wants you to create new views. I guess this is one of the missing parts.

New views are optional. The wording in the issue is "as needed," meaning we may not need them. 

The goal is to have a fully functional accounts app with only Django templates and views to remove the Backbone code. Implementation details are at your discretion.

**Comment by AbhijithGanesh** [comment]
This PR has a fully functional accounts app right? I've removed the backbone code too  I guess. If I haven't please request a review, I will make changes.

**Comment by brylie** [comment]
@AbhijithGanesh, I've created a new issue to list all of the code changes needed for `account.html`:

https://github.com/CiviWiki/OpenCiviWiki/issues/1199

Would you like to focus just on the single `account.html` file?

**Comment by AbhijithGanesh** [comment]
Yes I shall take note of this and focus on a single file

File: project/accounts/templates/accounts/user-setup.html [hunk 1]
```diff
@@ -8,7 +8,6 @@
 <link type="text/css" rel="stylesheet/less" href="{% static "less/setup.less" %}"/>
 {% endblock extra_css %}
 
-{% block backbone_template %}
```
Comment (brylie) [inline project/accounts/templates/accounts/user-setup.html:11]: Does the user setup wizard still work with this `block` tag removed?

**Review by brylie — COMMENTED** [review]
[deleted]

**Comment by AbhijithGanesh** [comment]
@brylie Closing this issue since It has been distributed.

**Pull request closed without merging**